packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/ProcessCameraProviderHostApiImpl.java

```diff
@@ -84,4 +91,52 @@ public List<Long> getAvailableCameraInfos(@NonNull Long identifier) {
     }
     return availableCamerasIds;
   }
+
+  /**
+   * Binds specified {@code UseCase}s to the lifecycle of the {@code LifecycleOwner} that
+   * corresponds to this instance and returns the instance of the {@code Camera} whose lifecycle
+   * that {@code LifecycleOwner} reflects.
+   */
+  @Override
+  public Long bindToLifecycle(
+      @NonNull Long identifier,
+      @NonNull Long cameraSelectorIdentifier,
+      @NonNull List<Long> useCaseIds) {
+    ProcessCameraProvider processCameraProvider =
+        (ProcessCameraProvider) instanceManager.getInstance(identifier);
```

Comment:
Could you wrap `instanceManager.getInstance(..)` with `Objects.requireNonnull(...)`. It helps with debugging and removes the lint warning. Same for the line below and other apis.

Reply:
cc @gmackall just a heads up about this